addons/mail/static/src/models/partner/partner.js

```diff
@@ -83,7 +83,7 @@ function factory(dependencies) {
          * @param {string} param0.keyword
          * @param {integer} [param0.limit=10]
          */
-        static async imSearch({ callback, keyword, limit = 10 }) {
+        static async imSearch({ callback, keyword, limit = 10, isMassMailing = false }) {
```

Comment:
I would not add this extra param now, this is a new feature while we are making a bugfix here.
I posted comment on the task.